packages/react/src/Pagination/model.ts

```diff
@@ -158,7 +158,7 @@ export function buildComponentData(
       key = 'page-prev'
       content = 'Previous'
       if (page.disabled) {
-        Object.assign(props, {as: 'button', 'aria-disabled': 'true'})
+        Object.assign(props, {rel: 'prev', as: 'button', 'aria-disabled': 'true'})
```

Comment:
I've added these in to make the hiding/showing of the Previous/Next text simpler. I believe these attributes should be present regardless as the buttons are still `prev`/`next` buttons irrespective of whether they are disabled or not.